proto/swapi.proto

```diff
@@ -267,3 +270,13 @@ message ListPeopleRequest {
 message ListPeopleResponse {
   repeated Person people = 1;
 }
+
+message StarshipAction {
+  Starship starship = 1;
+  enum Action {
```

Comment:
Did you mean to add a field as well?

Reply:
Yeah this is a WIP PR. I'll add a label.